### ISSUE TITLE
Custom Field not populating in correct Sequence.

### ISSUE THREAD

**Issue by suyashphadtare** [indictranstech/fedex_integration#1]
After bench migrate, custom fields are not populating in correct order.
bench migrate command needs to be done 4 times, then custom fields are appearing in proper order.

**Comment by erpnextdeveloper** [comment]
yes same issue custom fields not populated after bench migrate many times...
